examples/blend/main.rs

```diff
@@ -16,7 +16,7 @@
 extern crate gfx;
 extern crate gfx_app;
 extern crate image;
-extern crate winit;
+extern crate glutin;
```

Comment:
why is this changed to `glutin`?

Reply:
Since `glutin` itself bundles `winit` (and re-exports it) I thought it might be simpler to just use the `glutin` dependency instead.

I've removed `winit` as a direct dependency (except in `src/window/dxgi`, `src/window/vulcan` and `src/window/metal` where it's required). My reasoning was that it's easier to ensure we have the same version of `winit` across the entire repository if we use the `winit` that's re-exported by `glutin`.

If we add the `winit` dependency instead, then we have to ensure that it's the same version as the `winit` exported by `glutin` anyway.

Reply:
Happy to change all these back to `winit` and re-include it as well - that's an easy change. You can see what it would look like here: https://github.com/gfx-rs/gfx/compare/v0.17...acheronfail:chore/update-dependencies-with-winit